docs/source-app/glossary/environment_variables.rst

```diff
@@ -24,4 +24,4 @@ The environment variables are available in all flows and works, and can be acces
     print(os.environ["BAZ"])  # FAZ
 
 .. note::
-	Environment variables are currently not encrypted.
+	Environment variables are not encrypted. For sensitive values, we recommend using :ref:`Encrypted Secrets <secrets>`.
```

Comment:
```suggestion
	Environment variables are not encrypted. For sensitive values, we recommend using :ref:`Encrypted Secrets <_secrets>`.
```

Maybe this?

Reply:
I don't think that this is the solution, because the Sphinx docs indicate that the leading `_` isn't needed when using `:ref:`. Also, the other uses of `:ref:` in the docs code don't have leading underscores.

Reply:
Yeah, you are right. Looking into it.